codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/error/ServiceErrorGenerator.kt

```diff
@@ -191,7 +223,53 @@ class ServiceErrorGenerator(
                 rust("${sym.name}(#T),", sym)
             }
             docs("An unexpected error occurred (e.g., invalid JSON returned by the service or an unknown error code).")
+            rust(
+                "##[deprecated(note = \"Don't directly match on `Unhandled`. Instead, match on `_` and use " +
```

Comment:
same

Reply:
these deprecation messages should probably be centralized in codegen & include an explanation about why